bodhi/tests/server/consumers/test_masher.py

```diff
@@ -39,6 +39,7 @@
     UpdateRequest, UpdateStatus, UpdateType, User, ModuleBuild, ContentType, Package)
 from bodhi.server.util import mkmetadatadir, transactional_session_maker
 from bodhi.tests.server import base, populate
+import six
```

Comment:
PEP-8 import style would group ```six``` with the other third party imports, like ```mock``` above.

Reply:
Didn't know that, thanks. Fixed in the latest commit.